src/admin/utils/fieldTypes.jsx

```diff
@@ -515,6 +522,7 @@ const simple = {
             { const: "number", title: "Float" },
           ],
         },
+        tooltip: extra.optionsSchema.tooltip,
```

Comment:
Please move the tooltip up, here and in other fields, so that it appears directly under title and description (i.e. directly after `...common.optionsSchema`)